pkg/proxy/proxy.go

```diff
@@ -55,6 +54,13 @@ func getNextID() int64 {
 	return atomic.AddInt64(&idCounter, 1)
 }
 
+type DepInterface interface {
```

Comment:
Remove interface from the name of the type

Reply:
Modified